pkg/query-service/app/http_handler.go

```diff
@@ -2456,16 +2456,6 @@ func (ah *APIHandler) GetIntegration(
 		return
 	}
 
-	// Add connection status details.
-	connectionStatus, apiErr := ah.calculateConnectionStatus(
-		r.Context(), integration.ConnectionTests,
-	)
-	if apiErr != nil {
-		RespondError(w, apiErr, "Failed to calculate integration connection status")
-		return
-	}
-	integration.ConnectionStatus = connectionStatus
-
```

Comment:
Why is this being removed? It would be nice to know if any data was sent recently for the selected integration